pkg/sqlite/directory.go

```diff
@@ -128,10 +182,56 @@ func (d *DirectoryLoader) LoadBundleWalkFunc(path string, f os.FileInfo, err err
 	return utilerrors.NewAggregate(errs)
 }
 
-// LoadBundle takes the directory that a CSV is in and assumes the rest of the objects in that directory
+// LoadPackagesWalkFunc attempts to unmarshal the file at the given path into a PackageManifest resource.
+// If unmarshaling is successful, the PackageManifest is added to the loader's store.
+func (d *DirectoryLoader) LoadPackagesWalkFunc(path string, f os.FileInfo, err error) error {
```

Comment:
Sorry just realized, was this whole function copied and pasted from a different part of the file? Why?

Reply:
yeah I'm not sure why git thinks I re-wrote it...